src/lib/zkapp.ts

```diff
@@ -1306,7 +1306,16 @@ Use the optional \`maxTransactionsWithActions\` argument to increase this number
               )
             )
           );
-      });
+      }
+
+      if (Circuit.inCheckedComputation()) {
```

Comment:
A recent update in the Mina repo changed how snarky behaves. It used to set `inCheckedComputation = true` after *any* contract compilation - that now changed for some reason